### DISCUSSION TITLE
ChatGPT's BASE_URL

### DISCUSSION THREAD

**Discussion by pretenderlu** [plateaukao/einkbro#298] [General]
Haha, thanks to the awesome author for this great software. I wanted to ask if einkbro's chatgpt can consider supporting a third-party BASE_URL?

**Comment by plateaukao** [comment]
I'll consider adding it in next version.

**Reply by pretenderlu** [reply]
Cool!

**Reply by plateaukao** [reply]
It's added in this commit. https://github.com/plateaukao/einkbro/commit/76e848d24c51362741078754c6dd8792f0ef8b1a

Will be released after 2024 new year. :)

**Reply by pretenderlu** [reply]
太酷啦！
